internal/plugins/helm/v1/init.go

```diff
@@ -208,7 +208,7 @@ func addInitCustomizations(projectName string) error {
 		return err
 	}
 
-	// Remove the webhook option for the componentConfig since webhooks are not supported by ansible
+	// Remove the webhook option for the componentConfig since webhooks are not supported by helm/ansible
```

Comment:
You can fully remove `ansible` since this is the helm plugin
```suggestion
	// Remove the webhook option for the componentConfig since webhooks are not supported by helm
```

Reply:
Hi @estroz Will do! I'll also remove ansible from the other comments in the same file as well.  Thanks for the review!